src/courtroom.cpp

```diff
@@ -1476,6 +1497,7 @@ void Courtroom::set_fonts(QString p_char)
   set_font(ui_vp_showname, "", "showname", p_char);
   set_font(ui_vp_message, "", "message", p_char);
   set_font(ui_ic_chatlog, "", "ic_chatlog", p_char);
+  set_font(ui_ic_chatlog_pinned, "", "ic_chatlog_pinned", p_char);
```

Comment:
add `ic_chatlog_pinned` to the description so it's known what should be added to themes